### ISSUE TITLE
Switching Tabs in the Fuzzer causes highlighted text to be ignored.

### ISSUE THREAD

**Issue by MooseTheCoder** [zaproxy/zaproxy#5511]
**Describe the bug**
Switching tabs in the Fuzzer menu causes the "Add" button to become disabled, even if text is highlighted.

**To Reproduce**
1. Open the "Fuzzer"
2. Highlight part of the header to fuzz, notice the "Add" button being enabled
3. Switch to the "Options" tab
4. Switch Back to the "Fuzz Locations" tab
5. Notice the "Add" button being disabled even though the text is still highlighted

**Expected behavior**
Switching tabs should keep the "Add" button active if text is highlighted

**Software versions**
 - ZAP: 2.8.0
 - Add-on: AdvFuzzer 11.0.0.
 - OS: Windows 10
 - Java: 1.8.0_221
 - Browser: N/A

**Errors from the zap.log file**
N/A

**Additional context**
N/A

**Comment by kingthorin** [comment]
I was able to confirm this. It seems to have to do with which component gets focus when the tabs are switched. (Or more specifically when the fuzz location tab is returned to).

If you tab 4 times so that the header panel (where your highlight is) has focus then the add button becomes active again.

**Comment by thc202** [comment]
Yes, the focus is required to know from which field (header or body) is the fuzz location going to be added (e.g. you can have highlights in header and body at the same time).

**Comment by kingthorin** [comment]
So the focused component needs to be cached somehow when the tab loses focus and then restored when it retains focus....????

**Comment by thc202** [comment]
It's already cached so we'd just need to restore focus, or, stop disabling the button and assume that the user knows what was the last field focused?

**Comment by kingthorin** [comment]
True
